test/Groups/Permutations.jl

```diff
@@ -0,0 +1,44 @@
+@testset "Examples.Permutations" begin
+  p = @perm (1,2)(3,4)(5,6)
+  @test p == cperm([1,2],[3,4],[5,6])
+  p = @perm (1,2)(3,4,5)(7,8,9)
+  @test p == cperm([1,2],[3,4,5],[7,8,9])
+
+  a=1;b=2;f=x->3*x + 2;
+  p = @perm (a,f(a),b)(a+1,b*2)
+  @test p == cperm([1,5,4,2])
+  
+  @test_throws ErrorException @perm (-1, 1)
+  @test_throws LoadError @perm "bla"
```

Comment:
Ahhh now I understand where your `LoadError` comes from: the problem is that `@perm` is evaluated right away, while reading the file, and so `@test_throws` has no chance of catching that error.

You need to defer the expansion of the macro `@perm` in order to test it adequately. This can be achieved via `eval` resp.  the `@eval` macro. Try this:
```suggestion
  @test_throws LoadError @eval @perm "bla"
```